src/main/java/frc/robot/commands/RollerDrive.java

```diff
@@ -0,0 +1,47 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.LowScoringSubsystem;
+
+public class RollerDrive extends CommandBase {
+  private LowScoringSubsystem lowScoringSubsystem;
+  private double speed;
+
+  public RollerDrive(LowScoringSubsystem lowScoringSubsystem, double speed) {
+    this.lowScoringSubsystem = lowScoringSubsystem;
+    this.speed = speed;
+    addRequirements(lowScoringSubsystem);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+  }
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+    SmartDashboard.putNumber("Roller Speed", speed);
+    lowScoringSubsystem.setRollerSpeed(speed * speed);
```

Comment:
If we're not using a controller joystick for input here we can probably skip this squared input and just use speed:

```suggestion
    lowScoringSubsystem.setRollerSpeed(speed);
```

If we are going to use a joystick we should add a deadband.